__tests__/spec/migrate.js

```diff
@@ -76,7 +77,7 @@ describe('migrate test prototype', () => {
   it('routes.js', () => {
     const routesFileContents = fs.readFileSync(path.join(appDirectory, 'routes.js'), 'utf8')
 
-    expect(routesFileContents).toEqual(
+    expect(normaliseContent(routesFileContents)).toEqual(normaliseContent(
```

Comment:
I would recommend creating using strings for the input rather than using files on the file system so we don't have surprise test failures if/when we change the routes file etc.

Reply:
These tests are comparing the files actually generated by the migrate process itself so we can't really use strings as we wouldn't be checking the output of the process.  As an aside, I've removed the normaliseContent function as this is unneccessary when using gitbash on windows.

Reply:
I've updated the titles of these tests to indicate we are testing the files generated by the migration.  Please note that the beforeAll for these tests performs the migration to a temporary location and the resulting files are checked seperately within each test to see if they have been updated, overwrittem, or deleted correctly.

Reply:
Great, thanks for explaining. :)